### ISSUE TITLE
When installed causing an unhandled exception on new database file creation

### ISSUE THREAD

**Issue by S-ed** [jnko/SimpleDatabaseBackup#2]
With this plugin installed I couldn't create new database. KeePass crashes with unhandled exception:
```
System.IO.FileNotFoundException: Could not find file "d:\test.kdbx"
```
Steps to reproduce:
1. Make new database (with any settings, any path)
2. Save it

**Comment by jnko** [comment]
Yes, I can confirm that.  I guess I already know the source. A fix with an updated version will be available shortly.

Thanks for reporting!

**Comment by jnko** [comment]
Just released version 1.0.5 which fixes the bug described above. Please update the plugin.
